<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditProfessor</class>
 <widget class="QDialog" name="EditProfessor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>727</width>
    <height>476</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit/Display Professor</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/*-----QWidget-----*/
QWidget
{
	background-color: qlineargradient(spread:repeat, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(27, 39, 50, 255),stop:1 rgba(47, 53, 74, 255));
	color: #000000;

}


/*-----QLabel-----*/
QLabel
{
	background-color: transparent;
	color: #c2c7d5;
	font-size: 17px;

}


/*-----QPushButton-----*/
QPushButton
{
	background-color: qlineargradient(spread:pad, x1:0, y1:0.511, x2:1, y2:0.511, stop:0 rgba(0, 172, 149, 255),stop:0.995192 rgba(54, 197, 177, 255));
	color: #fff;
	font-size: 16px;
	font-weight: bold;
	border: none;
	border-radius: 6px;
	padding: 5px;

horizontal stretch: expanding;

}


QPushButton::pressed
{
	background-color: qlineargradient(spread:pad, x1:0, y1:0.511, x2:1, y2:0.511, stop:0 rgba(0, 207, 179, 255),stop:1 rgba(70, 255, 230, 255));

}


/*-----QCheckBox-----*/
QCheckBox
{
	background-color: transparent;
	color: #fff;
	font-size: 10px;
	font-weight: bold;
	border: none;
	border-radius: 5px;

}


/*-----QCheckBox-----*/
QCheckBox::indicator
{
    color: #b1b1b1;
    background-color: #323232;
    border: 1px solid darkgray;
    width: 12px;
    height: 12px;

}


QCheckBox::indicator:checked
{
    image:url(&quot;./ressources/check.png&quot;);
	background-color: qlineargradient(spread:pad, x1:0, y1:0.511, x2:1, y2:0.511, stop:0 rgba(0, 172, 149, 255),stop:0.995192 rgba(54, 197, 177, 255));;
    border: 1px solid #607cff;

}


QCheckBox::indicator:unchecked:hover
{
    border: 1px solid #08b099;

}


QCheckBox::disabled
{
	color: #656565;

}


QCheckBox::indicator:disabled
{
	background-color: #656565;
	color: #656565;
    border: 1px solid #656565;

}


/*-----QLineEdit-----*/
QLineEdit
{
	background-color: #c2c7d5;
	color: #000;
	font-weight: bold;
	border: none;
	border-radius: 2px;
	padding: 3px;

}


/*-----QListView-----*/
QListView
{
	background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(50, 61, 80, 255),stop:1 rgba(44, 49, 69, 255));
	color: #fff;
	font-size: 12px;
	font-weight: bold;
	border: 1px solid #191919;
	show-decoration-selected: 0;

}


QListView::item
{
	color: #31cecb;
	background-color: #454e5e;
	border: none;
	padding: 5px;
	border-radius: 0px;
	padding-left : 10px;
	height: 42px;

}

QListView::item:selected
{
	color: #31cecb;
	background-color: #454e5e;

}


QListView::item:!selected
{
	color:white;
	background-color: transparent;
	border: none;
	padding-left : 10px;

}


QListView::item:!selected:hover
{
	color: #bbbcba;
	background-color: #454e5e;
	border: none;
	padding-left : 10px;

}


/*-----QTreeView-----*/
QTreeView 
{
	background-color: #232939;
	show-decoration-selected: 0;
	color: #c2c8d7;

}


QTreeView::item 
{
	border-top-color: transparent;
	border-bottom-color: transparent;

}


QTreeView::item:hover 
{
	background-color: #606060;
	color: #fff;

}


QTreeView::item:selected 
{
	background-color: #0ab19a;
	color: #fff;

}


QTreeView::item:selected:active
{
	background-color: #0ab19a;
	color: #fff;

}


QTreeView::branch:has-children:!has-siblings:closed,
QTreeView::branch:closed:has-children:has-siblings 
{
	image: url(://tree-closed.png);

}

QTreeView::branch:open:has-children:!has-siblings,
QTreeView::branch:open:has-children:has-siblings  
{
	image: url(://tree-open.png);

}


/*-----QTableView &amp; QTableWidget-----*/
QTableView
{
    background-color: #232939;
	border: 1px solid gray;
    color: #f0f0f0;
    gridline-color: #232939;
    outline : 0;

}


QTableView::disabled
{
    background-color: #242526;
    border: 1px solid #32414B;
    color: #656565;
    gridline-color: #656565;
    outline : 0;

}


QTableView::item:hover 
{
    background-color: #606060;
    color: #f0f0f0;

}


QTableView::item:selected 
{
	background-color: #0ab19a;
    color: #F0F0F0;

}


QTableView::item:selected:disabled
{
    background-color: #1a1b1c;
    border: 2px solid #525251;
    color: #656565;

}


QTableCornerButton::section
{
	background-color: #343a49;
    color: #fff;

}


QHeaderView::section
{
	color: #fff;
	border-top: 0px;
	border-bottom: 1px solid gray;
	border-right: 1px solid gray;
	background-color: #343a49;
    margin-top:1px;
	margin-bottom:1px;
	padding: 5px;

}


QHeaderView::section:disabled
{
    background-color: #525251;
    color: #656565;

}


QHeaderView::section:checked
{
    color: #fff;
    background-color: #0ab19a;

}


QHeaderView::section:checked:disabled
{
    color: #656565;
    background-color: #525251;

}


QHeaderView::section::vertical::first,
QHeaderView::section::vertical::only-one
{
    border-top: 1px solid #353635;

}


QHeaderView::section::vertical
{
    border-top: 1px solid #353635;

}


QHeaderView::section::horizontal::first,
QHeaderView::section::horizontal::only-one
{
    border-left: 1px solid #353635;

}


QHeaderView::section::horizontal
{
    border-left: 1px solid #353635;

}


/*-----QScrollBar-----*/
QScrollBar:horizontal 
{
    background-color: transparent;
    height: 8px;
    margin: 0px;
    padding: 0px;

}


QScrollBar::handle:horizontal 
{
    border: none;
	min-width: 100px;
    background-color: #56576c;

}


QScrollBar::add-line:horizontal, 
QScrollBar::sub-line:horizontal,
QScrollBar::add-page:horizontal, 
QScrollBar::sub-page:horizontal 
{
    width: 0px;
    background-color: transparent;

}


QScrollBar:vertical 
{
    background-color: transparent;
    width: 8px;
    margin: 0;

}


QScrollBar::handle:vertical 
{
    border: none;
	min-height: 100px;
    background-color: #56576c;

}


QScrollBar::add-line:vertical, 
QScrollBar::sub-line:vertical,
QScrollBar::add-page:vertical, 
QScrollBar::sub-page:vertical 
{
    height: 0px;
    background-color: transparent;

}

QComboBox
{
color:#c2c7d5;


	font-size: 14px;
}

QGroupBox{
color:#c2c7d5;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_12">
     <item>
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Choose a Professor to see details:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_Profs"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Professor Info:</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label_10">
            <property name="font">
             <font>
              <pointsize>-1</pointsize>
             </font>
            </property>
            <property name="text">
             <string>First Name:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_11">
            <property name="font">
             <font>
              <pointsize>-1</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Last Name:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_12">
            <property name="font">
             <font>
              <pointsize>-1</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Title:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_13">
            <property name="font">
             <font>
              <pointsize>-1</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Age:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_14">
            <property name="font">
             <font>
              <pointsize>-1</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Email:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_15">
            <property name="font">
             <font>
              <pointsize>-1</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Mobile:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_17">
            <property name="font">
             <font>
              <pointsize>-1</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Courses Teaching:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_18">
            <property name="font">
             <font>
              <pointsize>-1</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Add a Course:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_19">
            <property name="font">
             <font>
              <pointsize>-1</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Remove a Course:</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLineEdit" name="lineEdit_first">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_last">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_ttl">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_age">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="readOnly">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_email">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_mobile">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox_courses"/>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox_addcrs"/>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox_remcrs"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="save">
          <property name="text">
           <string>Save Changes</string>
          </property>
          <property name="autoDefault">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="delete_2">
          <property name="text">
           <string>Remove Course</string>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="delete_prof">
          <property name="text">
           <string>Delete Professor</string>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="ok">
          <property name="text">
           <string>Main Menu</string>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
